maestro-cli/src/main/java/maestro/cli/cloud/CloudInteractor.kt

```diff
@@ -257,10 +278,19 @@ class CloudInteractor(
             saveReport(reportFormat, passed, upload, reportOutputSink, testSuiteName)
         }
 
-        return if (!passed) {
-            1
-        } else {
+
+        return if (passed) {
+            PrintUtils.message("Process will exit with code 0 (SUCCESS)")
+            if (isCancelled) {
+                PrintUtils.message("* To change exit code on Cancellation, run maestro with this option: `maestro cloud --fail-on-cancellation=<true|false>`")
+            }
             0
+        } else {
+            PrintUtils.message("Process will exit with code 1 (FAIL)")
+            if (isCancelled && !containsFailure) {
+                PrintUtils.message("* To change exit code on Cancellation, run maestro with this option: `maestro cloud --fail-on-cancellation=<true|false>`")
```

Comment:
Huge nit: I don't think we should capitalize cancellation. We generally only capitalize `Flows` and `Uploads`.
```suggestion
                PrintUtils.message("* To change exit code on cancellation, run maestro with this option: `maestro cloud --fail-on-cancellation=<true|false>`")
```